src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,94 @@
+    private static final boolean ELEVATOR_ENCODER_PHASE = false;
+    private static final int
+            ELEVATOR_ENCODER_OFFSET = 5,
+            ANGLE_ENCODER_OFFSET = 5;
+    private static final AbsoluteSensorRangeValue ANGLE_ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+
+    static final double
+            P = 1,
+            I = 0,
+            D = 0;
+    static final PIDController PID_CONTROLLER = new PIDController(P, I, D);
+    private static final CANSparkMax
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final CANcoder ANGLE_ENCODER = new CANcoder(ANGLE_ENCODER_ID);
+    private static final TalonSRX ELEVATOR_ENCODER = new TalonSRX(ELEVATOR_ENCODER_ID);
+
+    static {
+        configureAngleEncoder();
+        configureElevatorEncoder();
+        configureAngleMotor(MASTER_ANGLE_MOTOR);
+        configureAngleMotor(FOLLOWER_ANGLE_MOTOR);
+        configureElevatorMotor(MASTER_ELEVATOR_MOTOR);
+        configureElevatorMotor(FOLLOWER_ELEVATOR_MOTOR);
+    }
+
+    public enum ArmState {
+        ANGLE(new Rotation2d(5)),
+        ELEVATOR_POSITION(new Rotation2d(7));
+
+        final Rotation2d elevatorPosition;
+
+        ArmState(Rotation2d elevatorPosition) {
+            this.elevatorPosition = elevatorPosition;
+        }
+    }
+
+    private static void configureElevatorMotor(CANSparkMax motor) {
+        motor.restoreFactoryDefaults();
+        motor.setIdleMode(ELEVATOR_IDLE_MODE);
+        motor.setInverted(inverted);
+        motor.enableVoltageCompensation(VOLTAGE_COMPANSATION_SATURATION);
+    }
+
+    private static void configureAngleMotor(CANSparkMax motor) {
+        motor.restoreFactoryDefaults();
+        motor.setIdleMode(ANGLE_IDLE_MODE);
+        motor.setInverted(inverted);
+        motor.enableVoltageCompensation(VOLTAGE_COMPANSATION_SATURATION);
+    }
+
+    private static void configureAngleEncoder() {
+        CANcoderConfiguration angleEncoderConfig = new CANcoderConfiguration();
+        angleEncoderConfig.MagnetSensor.MagnetOffset = ANGLE_ENCODER_OFFSET;
+        angleEncoderConfig.MagnetSensor.AbsoluteSensorRange = ANGLE_ENCODER_RANGE;
+        angleEncoderConfig.MagnetSensor.SensorDirection = angleEncoderDirection;
+        ANGLE_ENCODER.getConfigurator().apply(angleEncoderConfig);
+    }
+
+    private static void configureElevatorEncoder() {
+        ELEVATOR_ENCODER.configFactoryDefault();
+        ELEVATOR_ENCODER.setSelectedSensorPosition(ELEVATOR_ENCODER.getSelectedSensorPosition() - ELEVATOR_ENCODER_OFFSET);
```

Comment:
Use the method from the class `Conversions` called `offsetRead` to calculate the offset